test/extended/util/annotate/rules.go

```diff
@@ -94,6 +94,14 @@ var (
 			`\[sig-cli\] Kubectl Port forwarding With a server listening on localhost should support forwarding over websockets`,
 			`\[sig-cli\] Kubectl Port forwarding With a server listening on 0.0.0.0 should support forwarding over websockets`,
 			`\[sig-node\] Pods should support remote command execution over websockets`,
+
+			// These tests are flacky and require internet access
+			// See https://bugzilla.redhat.com/show_bug.cgi?id=2019375
+			`\[sig-builds\]\[Feature:Builds\] build can reference a cluster service with a build being created from new-build should be able to run a build that references a cluster service`,
+			`\[sig-builds\]\[Feature:Builds\] oc new-app should succeed with a --name of 58 characters`,
+			`\[sig-network\] DNS should resolve DNS of partial qualified names for services`,
+			`\[sig-arch\] Only known images used by tests`,
+			`\[sig-network\] DNS should provide DNS for the cluster`,
```

Comment:
at least some of these seem like tests that should work in a proxy environment, has the investigation been done to understand why they are not?

i'm ok w/ disabling them while the investigation is done (via new bugs) but i'm not so sure we should just be disabling them w/o tracking a plan to fix them.

Reply:
@bparees I've created a BZ and assigned it to the right team: https://bugzilla.redhat.com/show_bug.cgi?id=2033628
I'll keep an eye and hopefully at some point remove these tests fro that list, so we can exercise them if it makes sense.

Thanks

Reply:
fyi it's probably going to end up needing more than one bug since these tests are owned by different teams